data/coaching.yaml

```diff
@@ -117,7 +117,7 @@
     - Miriam Nisbet
   actions:
     - label: More Info
-      link: freedom-of-information-and-foia-project-coaching-breaking-down-the-walls-and-opening-up-communications-detail.html
+      link: freedom-of-information-project-coaching-detail.html
```

Comment:
It's weird that these links are manually entered in the YAML rather than auto-generated by the build system. Filed #6 to address this.